qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/AppointmentBlocksPage.java

```diff
@@ -19,6 +19,7 @@
 public class AppointmentBlocksPage extends Page {
 
 	public static final By DELETE = By.linkText("Delete");
+	public static final By CURRENT_DAY_BUTTON = By.xpath("//*[@id=\"calendar\"]/table/tbody/tr/td[1]/span[5]");
```

Comment:
How about using another locator rather than xpath?

Reply:
> How about using another locator rather than xpath?

@kdaud  that button has a generic class name and it has not ID. I had initially used css selector and @sherrif10 advised me to use an xpath. I will continue looking into it to see the best selector I could come up with. Thanks.

Reply:
@Irenyak1 checkout the comment [here](https://github.com/openmrs/openmrs-contrib-qaframework/pull/425#discussion_r899801884)

Reply:
Sure, let me revert